dvc/repo/__init__.py

```diff
@@ -15,6 +15,7 @@
     OutputNotFoundError,
 )
 from dvc.path_info import PathInfo
+from dvc.template import PlotTemplates
```

Comment:
do we need this on a global repo level?

the same question for the `template.py` file itself ... do we need to have it on the dvc.* level?